### PR TITLE
Wip fix releasing

### PR DESCRIPTION
**Pull request by ktoso**
Follow up to: https://github.com/akka/akka-http/pull/409

### CODE REVIEW

File: project/Publish.scala [hunk 1]
```diff
@@ -14,7 +14,6 @@ object Publish extends AutoPlugin {
   override def requires = sbtrelease.ReleasePlugin
 
   override lazy val projectSettings = Seq(
-    crossPaths := false,
```
Comment (ktoso) [inline project/Publish.scala]: this is the actual fix. I confirmed this is enough to make things work.

**Comment by jrudolph** [comment]
LGTM

**Comment by 2m** [comment]
Just noticed that having this way of adding common settings confuses sbt about settings value in aggregated projects. For some reason this applies only to `organization`, `organizationName`, `organizationHomepage` and `crossVersion` keys:

```
akka-http-root > show crossVersion
[error] Ambiguous keys: {.}/*:crossVersion, *:crossVersion
[error] show crossVersion
[error]
```

This is with an autoplugin for common settings:

```
akka-http-root > show crossVersion
[info] akka-http-spray-json/*:crossVersion
[info]  Binary
[info] akka-http-xml/*:crossVersion
[info]  Binary
[info] akka-http-marshallers-scala/*:crossVersion
[info]  Binary
[info] akka-http-jackson/*:crossVersion
[info]  Binary
[info] akka-http-marshallers-java/*:crossVersion
[info]  Binary
[info] akka-parsing/*:crossVersion
[info]  Binary
[info] docs/*:crossVersion
[info]  Binary
[info] akka-http/*:crossVersion
[info]  Binary
[info] akka-http-core/*:crossVersion
[info]  Binary
...
```

**Comment by jonas** [comment]
However, it looks like the settings are applied correctly when lising using `akka-http-root` instead pf "default root":

```
akka-http-root > show akka-http-root/crossVersion
[info] akka-http-testkit/*:crossVersion
[info] 	Binary
[info] akka-http-core/*:crossVersion
[info] 	Binary
[info] akka-http-jackson/*:crossVersion
[info] 	Binary
[info] akka-http-marshallers-java/*:crossVersion
[info] 	Binary
[info] docs/*:crossVersion
[info] 	Binary
[info] akka-http-spray-json/*:crossVersion
[info] 	Binary
[info] akka-http-xml/*:crossVersion
[info] 	Binary
[info] akka-http-marshallers-scala/*:crossVersion
[info] 	Binary
[info] akka-http/*:crossVersion
[info] 	Binary
[info] akka-parsing/*:crossVersion
[info] 	Binary
[info] akka-http-tests/*:crossVersion
[info] 	Binary
[info] akka-http-root/*:crossVersion
[info] 	Binary
```

**Comment by jonas** [comment]
If this fix is enough to release RC1 I can look into the problem with ambiguous settings in the root project since I introduced the`inThisBuild` syntax.

Going with an autoplugin would help to simplify build.sbt and be useful if the plan is to have Akka-wide SBT plugins, but that's maybe more a discussion for akka-meta.

**Comment by ktoso** [comment]
_2.11 is shipped btw:
http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22akka-http_2.11%22

-- 
Konrad `ktoso` Malawski
Akka <http://akka.io> @ Lightbend <http://lightbend.com>

On 19 October 2016 at 15:23:27, Jonas Fonseca (notifications@github.com)
wrote:

If this fix is enough to release RC1 I can look into the problem with
ambiguous settings in the root project since I introduced theinThisBuild
syntax.

Going with an autoplugin would help to simplify build.sbt and be useful if
the plan is to have Akka-wide SBT plugins, but that's maybe more a
discussion for akka-meta.

—
You are receiving this because you modified the open/close state.
Reply to this email directly, view it on GitHub
<https://github.com/akka/akka-http/pull/410#issuecomment-254810843>, or mute
the thread

**Comment by jonas** [comment]
Followed up with #413

**Pull request merged**